pyfr/backends/openmp/types.py

```diff
@@ -37,12 +37,11 @@ def hdata(self):
         return self.data
 
 
-class OpenMPMatrixRSlice(base.MatrixRSlice):
-    @lazyprop
-    def data(self):
-        return self.parent.data[self.p:self.q]
+class OpenMPMatrixSlice(base.MatrixSlice):
+    def _init_data(self, mat):
+        return mat.data[self.ra:self.rb, self.ca:self.cb]
```

Comment:
Flux AA seems to break the implementation here with attribute error
AttributeError: 'OpenMPMatrix' object has no attribute 'data'.

Traces back to base/elements.py line 183
self._scal_fpts = self._scal_fqpts.slice(0, nfpts)

Looks like self.data of the OpenMP matrix to which we are applying the slice is not initialised.